### PR TITLE
DX-1428 Revisit Prices

### PR DESCRIPTION
**Pull request by arebra**
Fleshed out the prices info and type info.

### CODE REVIEW

**Review by NullableInt — APPROVED** [review]
Thank you! This makes it more clear to me what this `prices` is 😃 🥰

File: _includes/prices.md [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 ## Prices
 
-The `prices` resource lists the prices related to a specific payment.
+The `prices` resource lists the prices related to a specific payment. In short,
+ it is where you enter the payment's amount. It consists of the payment's `id`
+ and the `priceList`, which again contains the payment's `type`, `amount` and
+ `vatAmount`.
```
Comment (asbjornu) [inline _includes/prices.md:8]:
```suggestion
it is where you enter the payment's amount. It consists of the payment's `id`
and the `priceList`, which again contains the payment's `type`, `amount` and
`vatAmount`. 
```

File: _includes/prices.md [hunk 2]
```diff
@@ -54,26 +62,31 @@ Content-Type: application/json
...
-to a card brand or bank respectively.
+Each payment instrument have one or more prices object types. Usually there is
+only one, whichs corresponds with the name of the payment instrument, like
+`Vipps`, `Swish` or `Mobilepay`. 
+
+The most common occurence of several object types is for card payments. While it
+is possible to group them all as `Creditcard`, you can also differentiate on
+card types as shown in the example above. This is useful if certain card brands
+have additional fees, others have discounts or something similar. If you do
+differentiate, you need to add all accepted card brand as a separate object
```
Comment (asbjornu) [inline _includes/prices.md:73]:
```suggestion
differentiate, you need to add all accepted card brands as a separate object
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Excellent!

**Review by asbjornu — APPROVED** [review]
👏🏼 🎉

**Pull request merged**